recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```

Comment:
Done, thanks @jakirkham , I was not aware of this syntax, seems to be undocumented on https://conda.io/docs/building/meta-yaml.html#preprocessing-selectors .